calm/dsl/cli/bp_commands.py

```diff
@@ -217,8 +217,19 @@ def create_blueprint_command(bp_file, name, description):
     default=False,
     help="Ignore runtime variables and use defaults",
 )
+@click.option(
```

Comment:
1. Rename the parameter name to `launch_params`
2. Use python file (instead of json_file to send those parameters). (use `runtime_vars` variable in it to get runtime variables)
    i.e. param.py file
`runtime_vars = [
         {...}..
],

runtime_substrates = [
]              # ignore runtime_substrates variable for now
`

> At EOD, we will also validate this file for proper structure to have only desired variables(Ignore now)